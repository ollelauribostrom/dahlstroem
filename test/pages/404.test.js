import React from 'react';
import { shallow } from 'enzyme';

import NotFoundPage from '../../src/pages/404';

describe('Tests for 404 page', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
  });
});
