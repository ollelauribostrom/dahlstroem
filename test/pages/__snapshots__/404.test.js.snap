// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for 404 page renders correctly 1`] = `
<Layout
  color={
    Object {
      "hex": "#000",
    }
  }
  menuBackground={
    Object {
      "hex": "#000",
    }
  }
  menuColor={
    Object {
      "hex": "#fff",
    }
  }
  width={810}
>
  <SEO
    keywords={Array []}
    lang="en"
    meta={Array []}
    title="404: Not found"
  />
  <h1>
    NOT FOUND
  </h1>
  <p>
    You just hit a route that doesn't exist... the sadness.
  </p>
</Layout>
`;
